docs/USER_MANAGEMENT_API_V2.md

```
@@ -0,0 +1,729 @@
+## Merchant Endpoints
+
+### 1. List All Merchants
+
+```http
+GET /api/breeze-buddy/users/merchants
+```
+
+**Access:** Admin only
+
+**Query Parameters:**
+| Parameter | Type | Default | Range | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)